### PR TITLE
Add own solutions

### PR DESCRIPTION
**Pull request by clarfonthey**
After taking forever to finally get to 100%ing this game, I decided to see which of my solutions were unique among these. Most of them weren't really unique ideas, but I took the opportunity to mark similar ones as obsolete.

Commits are separated so they're easier to parse, and the most notable ones are:

* Solutions were marked as non-executable (https://github.com/atesgoral/hrm-solutions/pull/332/commits/ec0dd4719ae2e765499666a0b92fc921bec4e90f)
* Trailing whitespace and newlines were removed from solutions (https://github.com/atesgoral/hrm-solutions/pull/332/commits/d15b31b1d6f36507f625e8d3cb0b9f150b8c9d73)
* Solution for 20-Multiplication-Workshop (https://github.com/atesgoral/hrm-solutions/pull/332/commits/319600e86210e5b85b3439990c9a32e9373191c2)
* Solution for 22-Fibonacci-Visitor (https://github.com/atesgoral/hrm-solutions/pull/332/commits/21b4f649e0f9298d764e5a5a9d29062977457e23)
* Solution for 38-Digit-Exploder (https://github.com/atesgoral/hrm-solutions/pull/332/commits/2b0dfc3641c47606e3d2956c5f139c29be2cd7eb)
* Solution for 40-Prime-Factory (https://github.com/atesgoral/hrm-solutions/pull/332/commits/8b3df699facaa5f1542dd3302d3d847ba9ea1637)
* Solution(s) for 41-Sorting-Floor (https://github.com/atesgoral/hrm-solutions/pull/332/commits/bfa4682306eb9ab923be3fb9fcc9859917fb8a9e)
* Solution had an invalid name (https://github.com/atesgoral/hrm-solutions/pull/332/commits/390063b989480fb9027c29614f4c540859cfe071)

I figure you don't really have the time/motivation to review the changes in detail, but I figured I'd submit my stuff anyway since I like this project.

I had to do an `npm update` in order to get `npm test` to run, but I didn't include the package-lock changes in the PR since I didn't want to break anything.

### CODE REVIEW

File: solutions/21-Zero-Terminated-Sum-10.72/10.88.obsolete-clarfonthey.asm [hunk 1]
```diff
@@ -0,0 +1,29 @@
+-- HUMAN RESOURCE MACHINE PROGRAM --
+-- 21-Zero-Terminated-Sum - SIZE 10/10 - SPEED 88/72 --
+
+-- This solution is superseded by 10.68-skwasjer.asm being funcitonally same
```
Comment (atesgoral) [inline solutions/21-Zero-Terminated-Sum-10.72/10.88.obsolete-clarfonthey.asm:4]:
```suggestion
-- This solution is superseded by 10.68-skwasjer.asm being functionally same
```

File: solutions/21-Zero-Terminated-Sum-10.72/11.68.obsolete-clarfonthey.asm [hunk 2]
```diff
@@ -0,0 +1,31 @@
+-- HUMAN RESOURCE MACHINE PROGRAM --
+-- 21-Zero-Terminated-Sum - SIZE 11/10 - SPEED 68/72 --
+
+-- This solution is superseded by 10.68-skwasjer.asm being funcitonally same
```
Comment (atesgoral) [inline solutions/21-Zero-Terminated-Sum-10.72/11.68.obsolete-clarfonthey.asm:4]:
```suggestion
-- This solution is superseded by 10.68-skwasjer.asm being functionally same
```

File: solutions/22-Fibonacci-Visitor-19.156/16.126.obsolete-skwasjer.asm [hunk 3]
```diff
@@ -1,15 +1,18 @@
 -- HUMAN RESOURCE MACHINE PROGRAM --
 -- 22-Fibonacci-Visitor - SIZE 16/19 - SPEED 126/156 --
 
+-- This solution is superseded by 15.126-sniperrifle2004.asm being funcitonally
```
Comment (atesgoral) [inline solutions/22-Fibonacci-Visitor-19.156/16.126.obsolete-skwasjer.asm:4]:
```suggestion
-- This solution is superseded by 15.126-sniperrifle2004.asm being functionally
```

File: solutions/22-Fibonacci-Visitor-19.156/17.148.obsolete-tuxuin.asm [hunk 4]
```diff
@@ -1,18 +1,18 @@
 -- HUMAN RESOURCE MACHINE PROGRAM --
 -- 22-Fibonacci-Visitor - SIZE 17/19 - SPEED 148/156 --
 
--- This solution is superseded by 16.126-skwasjer.asm being funcitonally same
--- with further optimization.
+-- This solution is superseded by 15.126-sniperrifle2004.asm being funcitonally
```
Comment (atesgoral) [inline solutions/22-Fibonacci-Visitor-19.156/17.148.obsolete-tuxuin.asm:4]:
```suggestion
-- This solution is superseded by 15.126-sniperrifle2004.asm being functionally
```

File: solutions/22-Fibonacci-Visitor-19.156/18.154.obsolete-cowboy.asm [hunk 5]
```diff
@@ -1,18 +1,18 @@
 -- HUMAN RESOURCE MACHINE PROGRAM --
 -- 22-Fibonacci-Visitor - SIZE 18/19 - SPEED 154/156 --
 
--- This solution is superseded by 17.148-tuxuin.asm being functionally same with
--- further optimization.
+-- This solution is superseded by 15.126-sniperrifle2004.asm being funcitonally
```
Comment (atesgoral) [inline solutions/22-Fibonacci-Visitor-19.156/18.154.obsolete-cowboy.asm:4]:
```suggestion
-- This solution is superseded by 15.126-sniperrifle2004.asm being functionally
```

File: solutions/22-Fibonacci-Visitor-19.156/19.156.obsolete-18111398.asm [hunk 6]
```diff
@@ -1,8 +1,11 @@
 -- HUMAN RESOURCE MACHINE PROGRAM --
 -- 22-Fibonacci-Visitor - SIZE 19/19 - SPEED 156/156 --
 
+-- This solution is superseded by 15.126-sniperrifle2004.asm being funcitonally
```
Comment (atesgoral) [inline solutions/22-Fibonacci-Visitor-19.156/19.156.obsolete-18111398.asm:4]:
```suggestion
-- This solution is superseded by 15.126-sniperrifle2004.asm being functionally
```

File: solutions/22-Fibonacci-Visitor-19.156/19.177.obsolete-clarfonthey.asm [hunk 7]
```diff
@@ -0,0 +1,54 @@
+-- HUMAN RESOURCE MACHINE PROGRAM --
+-- 22-Fibonacci-Visitor - SIZE 19/19 - SPEED 177/156 --
+
+-- This solution is superseded by 15.126-sniperrifle2004.asm being funcitonally
```
Comment (atesgoral) [inline solutions/22-Fibonacci-Visitor-19.156/19.177.obsolete-clarfonthey.asm:4]:
```suggestion
-- This solution is superseded by 15.126-sniperrifle2004.asm being functionally
```

**Review by atesgoral — APPROVED** [review]
What a fantastic contribution. Thank you!

I've reviewed it commit by commit.

Just some nitpicky typo fixes.

**Comment by atesgoral** [comment]
@clarfonthey you're more than welcome to become a maintainer after this much 💅

**Comment by clarfonthey** [comment]
No worries, this is such an old game I honestly wasn't expecting you to even get to this that soon. 🙏🏻

**Pull request merged**